File: src/main/java/dev/swe573/whatsthis/dto/TagDto.java
package dev.swe573.whatsthis.dto;

import lombok.Data;

@Data
public class TagDto {
    private String id;
    private String label;
    private String description;
    private String wikiUrl;
    private String wikipediaUrl;

    public TagDto (String id, String label, String description, String wikiUrl, String wikipediaUrl) {
        this.id = id;
        this.label = label;
        this.description = description;
        this.wikiUrl = wikiUrl;
        this.wikipediaUrl = wikipediaUrl;
    }

}

File: mobile/eurekaNexus/app/src/main/java/com/yusuf/eurekanexus/data/model/TagDto.kt
package com.yusuf.eurekanexus.data.model

import com.google.gson.annotations.SerializedName

data class TagDto(
    @SerializedName("label") val label: String,
    @SerializedName("description") val description: String,
    @SerializedName("wikiUrl") val wikiUrl: String,
    @SerializedName("wikipediaUrl") val wikipediaUrl: String
)